bouncer/app.py

```diff
@@ -53,5 +55,21 @@ def create_app(_=None, **_settings):  # pragma: nocover
     }
     config.include("bouncer.search")
     config.include("bouncer.views")
+
+    # Enable Sentry's "Releases" feature, see:
+    # https://docs.sentry.io/platforms/python/configuration/options/#release
+    #
+    # h_pyramid_sentry passes any h_pyramid_sentry.init.* Pyramid settings
+    # through to sentry_sdk.init(), see:
+    # https://github.com/hypothesis/h-pyramid-sentry?tab=readme-ov-file#settings
+    #
+    # For the full list of options that sentry_sdk.init() supports see:
+    # https://docs.sentry.io/platforms/python/configuration/options/
+    config.add_settings(
+        {
+            "h_pyramid_sentry.init.release": get_version(),
```

Comment:
The other option is to just set the `SENTRY_RELEASE` envvar in Elastic Beanstalk but I don't think we have a good way to automatically set that envvar based on the release. Bouncer already has this `get_version()` function that seems to be [based on git-archive substitutions](https://github.com/hypothesis/bouncer/blob/e3e61fb340a7f636a063f32cff7758d3b572b742/bouncer/_version.py#L9-L16), so let's give that a try.